src/chrome/content/rules/domaindiscount24.xml

```diff
@@ -0,0 +1,19 @@
+<!--
+	Covers domaindiscount24.com and domaindiscount24.net
+
+	No HTTPS support:
+		* wiki.domaindiscount24.com
+-->
+<ruleset name="Domain Discount 24">
+	<target host="domaindiscount24.com" />
+	<target host="www.domaindiscount24.com" />
+	<target host="domaindiscount24.net" />
+	<target host="www.domaindiscount24.net" />
+	<target host="webmail.domaindiscount24.com" />
+	<target host="login.domaindiscount24.com" />
+
+  <securecookie host="^(www|webmail|login\.)?domaindiscount24\.com$" name=".+" />
```

Comment:
Oh, yes, thanks for noticing me.
Fixed with https://github.com/EFForg/https-everywhere/commit/f5b4cc05521053bec7c3a994983369ead303990c.